src/main/webapp/deletefile.html

```diff
@@ -26,7 +26,7 @@ <h2>Delete Configuration</h2>
       <a href="createfile.html">Create Configuration</a>
     </div><br>
     <div id="delete-search">
-      <form onSubmit="return deleteVendor(this)">
+      <form onSubmit="javascript:deleteVendor(this)">
```

Comment:
is the `this` parameter necessary? it may be, but I imagine it's not?

Reply:
I am passing in a form that has the vendor ID. If I don't pass it in, I won't be able to get the vendorID.